public/docs/ts/latest/guide/i18n.jade

```diff
@@ -0,0 +1,302 @@
+include ../_util-fns
+
+:marked
+  With internationalization, also known as i18n, we can display our website in multiple languages. There are two different
+  approaches to internationalization: with AoT compilation which requires a build step and a full reload when the language changes,
+  or with JiT which doesn't require a full reload and uses bindings to determine when a translation is needed.
+  
+  
+  
+  <a id="top"></a>
+  ## Table of contents
+  
+  [Different approaches to internationalization](#approaches-internationalization)
+  
+    * [JiT](#jit)
+    * [AoT](#aot)
+    
+  [Angular and i18n](#angular2-i18n)
+  
+    * [i18n directive](#i18n-directive)
+    * [i18n plural pipe](#i18n-plural-pipe)
+    * [i18n select pipe](#i18n-select-pipe)
+    
+  [Extract and use translations](#extract)
+  
+    * [Extract messages with ng-xi18n](#ng-xi18n)
+    * [Translate messages](#translate)
+    * [Use translations with the JiT approach](#use-jit)
+    * [Use translations with the AoT approach](#use-aot)
+
+  
+.l-main-section
+<a id="approaches-internationalization"></a>
+:marked
+  ## Different approaches to internationalization
+
+  
+a(id="jit")
+.l-main-section
+:marked
+  ### Just in time (JiT) approach
+  
+  The JiT approach relies on providing the translations to our application and binding them to our templates using directives, pipes and interpolations.
+  
+  Its main features are:
+    * Minimal server side support required:  The same version of the application code is served by the server.
+    However, the server must also serve translated message bundles back to the application or have tools that embed all
+    translations in the application code at build time.
+    * Our application must track all the pieces of the UI that need to be updated when the locale changes.
+    In addition, if the new language strings are being loaded over the network, this could take time and the UI needs to indicate this in some way to the user.
+    * Allows one to support multiple languages in the same view.
```

Comment:
I think there is a confusion:
dynamic/static vs JiT/AoT

Angular always use the static approach  for both AoT and JiT

Usage of pipes should be discouraged. They should only be used by the compiler to transform codes.

If you use the pipes you basically can not i18n your app because different langs use different plural category